src/cli/init/index.ts

```diff
@@ -20,29 +20,38 @@ import {
   tsConfig,
 } from "./templates";
 import { createDir, sanitizeName, write } from "./utils";
-import { confirm, walkthrough } from "./walkthrough";
+import { confirm, PromptOptions, walkthrough } from "./walkthrough";
+import { ErrorList } from "../../lib/errors";
 
 export default function (program: RootCmd) {
+  let response = {} as PromptOptions;
+  let pkgOverride = "";
   program
     .command("init")
     .description("Initialize a new Pepr Module")
-    // skip auto npm install and git init
-    .option("--skip-post-init", "Skip npm install, git init and VSCode launch")
-    .action(async opts => {
-      let pkgOverride = "";
-
-      // Overrides for testing. @todo: don't be so gross with Node CLI testing
+    .option("--confirm", "Skip verification prompt before creating module")
+    .option("--description <string>", "Set a description!")
```

Comment:
```suggestion
    .option("--description <string>", "Set a description.")
```

https://github.com/defenseunicorns/pepr/blob/7e7b03eab0c1f367be7bd2a484f0e5fec7088fe6/src/cli/build.ts#L62

Using the rest of the commands for reference, the exclamation doesn't look right for a regular CLI command